client/netdb.c

```diff
@@ -22,6 +23,12 @@
 #define DATABASE            (g_xdag_testnet ? "netdb-testnet.txt" : "netdb.txt")
 #define DATABASEWHITE       (g_xdag_testnet ? "netdb-white-testnet.txt" : "netdb-white.txt")
 
+#define whitelist_url            "https://raw.githubusercontent.com/XDagger/xdag/master/client/netdb-white.txt"
+#define whitelist_url_testnet    "https://raw.githubusercontent.com/XDagger/xdag/master/client/netdb-white-testnet.txt"
+#define WHITE_URL                (g_xdag_testnet ? whitelist_url_testnet : whitelist_url)
```

Comment:
Can you implement the same for netdb.txt and netdb-testnet.txt? You can do it as a separate branch.

Reply:
OK, no problem, I will add netdb.txt and netdb-testnet.txt.

Reply:
netdb.txt is automatically propagated through network, so it makes sense to find and turn off this logic.

Reply:
@jonano614 
Checked the code of netdb.txt, it's updated through network. If there is invalid pool not in white-list connect to a valid pool, the valid pool used to white-list this invalid pool, the invalid pool will be in netdb.txt even it is removed from white-list... That's strange..... I will check in another branch.